synapse/storage/database.py

```diff
@@ -58,7 +58,7 @@
 from synapse.metrics.background_process_metrics import run_as_background_process
 from synapse.storage.background_updates import BackgroundUpdater
 from synapse.storage.engines import BaseDatabaseEngine, PostgresEngine, Sqlite3Engine
-from synapse.storage.types import Connection, Cursor
+from synapse.storage.types import Connection, Cursor, _Parameters
```

Comment:
If we're now importing `_Parameters` somewhere else can we rename it to `TransactionParameters` or something? It could also use a comment above it...

Reply:
Can do. Will probably call it `ExecuteParameters` or `QueryParameters`.

Reply:
Went for `SQLQueryParamters` in the end (as opposed to URL query parameters).